docs/architecture.md

```diff
@@ -2,65 +2,38 @@
+## ProjectBuilder Components
 
-The project builder itself is based on the same project structure like the lireparp template. So the following description is valid for both. The project, based on JEE-standards, is build via maven and is parted in subprojects shown in the diagram below. For faster handling, an ant-file is provided. So, assumed ant, maven and JDK 1.8 are installed, you just need to execute ant setup_project on command line to install and start the project. By executing ant info you can find several other parts helping to deal with the project. The project consists of several subprojects depicted in the following diagrams.
-Subprojects of lirejarp template
+The project builder itself is based on the same project structure like the [starwit/reacthook-spring-template](https://github.com/starwit/reacthook-spring-template). The following description is valid for both. The project, based on Spring Boot is build via maven and is parted in subprojects shown in the diagram below.
 
-The following subprojects exist in every project built with lirejarp project builder:
-
-### persistence 
-This subproject contains classes for database access. Transactions are ensured through Enterprise Java Beans, if the database supports transactions.
+![Components projectBuilder](diagrams/projectbuilder-architecture-intern.drawio.svg)
 
+The following subprojects exist in every project built with projectBuilder:
+### application
+This subproject defines the over-all spring boot application. It contains spring boot application configuration and is needed to build the application.
+### persistence
+This subproject contains classes for database access. In this project, all database scripts are placed. Through flyway, these scripts are executed automatically during project start. Two databases has to be supported with the scripts: h2 (for tests) and mysql.
 ### rest
 This subproject enables the access from UI via RESTful webservices.
 ### webclient
-The UI logic is placed in the webclient project. ReactJS is used. The Frontend is utilized with [Material-UI](https://www.mui.com) and its design structure (mostly based on [JSS](https://cssinjs.org/?v=v10.8.1)).
-
-### packagewar
-The deployment of the project is done via war-file. It contains the projects persistence and rest as jar-files and the webclient. The project packagewar is resonsible for the build.
-
-![Component Diagram LireJarp](diagrams/lirejarpTemplate.png)
-
-## Additional projects in lirejarp project builder
-In lirejarp project builder, two additional projects are implemented:
+The UI logic is placed in the webclient project. React is used. The Frontend is utilized with [Material-UI](https://www.mui.com) and its design structure (mostly based on [JSS](https://cssinjs.org/?v=v10.8.1)).
```

Comment:
Cool information with JSS but I would probably link to the design tools on the mui page. Because things could change in the future...

Reply:
Die infos stammen von dir - kannst du gern ändern. Ich habe das glaube ich nur verschoben

Reply:
Okey... Soll ich das nochmal anpassen?

Reply:
ja - bitte gleich einchecken